### PR TITLE
add sortBy icons and indexes

### PR DESCRIPTION
**Pull request by kaligrafy**

### CODE REVIEW

File: packages/evolution-frontend/src/components/pageParts/validations/InterviewList.tsx [hunk 1]
```diff
@@ -122,9 +124,35 @@ const InterviewList = <CustomSurvey, CustomHousehold, CustomHome, CustomPerson>(
...
+                                </span>
+                                <a
+                                    className={`faIconRight${
+                                        sortBy.length === 1 &&
+                                        sortBy[0].id === column.accessor.toString() &&
+                                        sortBy[0].desc !== true
+                                            ? ' _active-background _blue'
+                                            : ''
+                                    }`}
+                                    onClick={() => handleSortingChange(column.accessor)}
```
Comment (tahini) [inline packages/evolution-frontend/src/components/pageParts/validations/InterviewList.tsx:138]: `handleSortingChange` changes the current sorting order of the accessor, but here you have an arrow that should be 'asc' and another that should be 'desc'. If you click twice on the down arrow, there should be no change. You may have to send a second parameter in handleSortingChange with the direction or sorting and don't update if the order is the same

File: packages/evolution-frontend/src/components/pageParts/validations/InterviewListComponent.tsx [hunk 2]
```diff
@@ -172,7 +174,14 @@ const InterviewListComponent: React.FunctionComponent<InterviewListComponentProp
                             title={props.t('admin:persons')}
                         />
                     </React.Fragment>
-                )
+                ),
+                label: props.t('admin:HouseholdSize'),
+                enableSortBy: true
+            },
+            {
+                accessor: 'responses.household.carNumber',
```
Comment (tahini) [inline packages/evolution-frontend/src/components/pageParts/validations/InterviewListComponent.tsx:182]: Add a TODO that says this should be project specific, like on line 133.

**Review by tahini — CHANGES_REQUESTED** [review]

File: packages/evolution-frontend/src/components/pageParts/validations/InterviewList.tsx (hunk 1 above)
Comment (kaligrafy) [reply packages/evolution-frontend/src/components/pageParts/validations/InterviewList.tsx]: fixed

File: packages/evolution-frontend/src/components/pageParts/validations/InterviewListComponent.tsx (hunk 2 above)
Comment (kaligrafy) [reply packages/evolution-frontend/src/components/pageParts/validations/InterviewListComponent.tsx]: removed car number completely, not needed unless someone asks for it.

**Pull request merged**